src/com/theironyard/novauc/CInputProgram.java

```diff
@@ -0,0 +1,27 @@
+package com.theironyard.novauc;
+
+import java.util.Scanner;
+
+class CInputProgram {
+    public static Scanner scannerln = new Scanner(System.in);
+    public static Scanner intInput = new Scanner(System.in);
+
+    public static void main(String[] args) throws Exception {
+        RWJson rwJson = new RWJson();
```

Comment:
Good Job D'Angelo.  Good program design. I like the private variables and breaking the read and write json up.